var/spack/repos/builtin/packages/parmetis/enable_external_metis.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ca945dd..1bf94e9 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -33,7 +33,7 @@ include_directories(${GKLIB_PATH})
+ include_directories(${METIS_PATH}/include)
+
+ # List of directories that cmake will look for CMakeLists.txt
+-add_subdirectory(${METIS_PATH}/libmetis ${CMAKE_BINARY_DIR}/libmetis)
++#add_subdirectory(${METIS_PATH}/libmetis ${CMAKE_BINARY_DIR}/libmetis)
```

Comment:
This is not enough. You have to add a `find_library(METIS_LIBRARY metis)` or something like this. ?In the current state it does not link properly.

> /home/richart/spack/opt/spack/linux-x86_64/gcc-4.4.7/binutils-2.25-xdwrsshxbacqpzliypzrrjicrgetxc7r/bin/ld: cannot find -lmetis`